src/DataStructures/Tensor/IndexType.hpp

```diff
@@ -43,6 +43,7 @@ struct Logical {};
 struct Grid {};
 struct Inertial : FrameIsPhysical {};
 struct Distorted {};
+struct ComovingFluid {};
```

Comment:
During the hydro call, we established that ComovingFluid is not a Frame as the tensors are in the inertial frame, describing quantities comoving with the fluid